build-tools/automation/azure-pipelines.yaml

```diff
@@ -378,6 +380,8 @@ stages:
         project: src/Mono.Android/Test/Mono.Android-Tests.csproj
         testResultsFiles: TestResult-Mono.Android_Tests-$(ApkTestConfiguration)-Aot.xml
         extraBuildArgs: /p:AotAssemblies=True /p:EnableLlvm=True
+        artifactName: Mono.Android_Tests-Signed.apk
+        artifactFolder: Aot
```

Comment:
Should this `artifactFolder` be `Aot+LLVM`?  Or do we not need to distinguish when LLVM was used?